doc/zmq.md

```diff
@@ -87,40 +87,69 @@ For instance:
                -zmqpubrawtx=ipc:///tmp/bitcoind.tx.raw \
                -zmqpubhashtxhwm=10000
 
-Each PUB notification has a topic and body, where the header
-corresponds to the notification type. For instance, for the
-notification `-zmqpubhashtx` the topic is `hashtx` (no null
-terminator). These options can also be provided in bitcoin.conf.
+Notification types correspond to message topics (details in next section). For instance,
```

Comment:
In commit "doc: Fix and clarify description of ZMQ message format" (7a93544cdcc2874a18a8b6d528a75e84ac007880)

Note: this is no longer mentioning topic strings do not include null terminators, bu this is probably a good thing. There shouldn't be a reason to think that they would include null terminators.